src/libmongoc/tests/unified/operation.c

```diff
@@ -476,63 +476,53 @@ bulk_op_append (mongoc_bulk_operation_t *bulk,
       if (!bson_parser_parse (parser, &request_doc, error)) {
          goto done;
       }
-      if (!mongoc_bulk_operation_insert_with_opts (
-             bulk, document, bson_parser_get_extra (parser), error)) {
-         goto done;
-      }
+
+      mongoc_bulk_operation_insert_with_opts (
+         bulk, document, bson_parser_get_extra (parser), error);
```

Comment:
The previous code made it impossible to evaluate `expectError` for these methods, since any error interrupted the test runner's execution (like a parsing error or unsupported operation). This change is consistent with how other public API methods are invoked.